source/usage-examples/find.txt

```diff
@@ -4,13 +4,16 @@ Find Multiple Documents
 
 .. default-domain:: mongodb
 
-You can query for multiple documents in a collection with
-``collection.find()``. The ``find()`` method uses a query document that you
-provide to match the subset of the documents in the collection that match the
-query. If you don't provide a query document (or if you provide an empty
-document), MongoDB returns all documents in the collection. For more
-information on querying MongoDB, see our
-:doc:`documentation on query documents </fundamentals/crud/query-document>`.
+You can query for multiple documents in a collection by passing a
+query document to the ``collection.find()`` method. A query document
+contains one or more query operators that apply to specific fields which
+determine the documents to include in the result set. For more
+information, see our :doc:`documentation on query documents </fundamentals/crud/query-document>`.
+
+.. note::
+
+  If you don't provide a query document (or if you provide an empty
+  document), MongoDB returns all documents in the collection. 
```

Comment:
Suggestion: I think this content should not be broken out of the paragraph. I don't have supporting evidence for this unfortunately, but in previous reviews I was informed that readers are less likely to read content in notes. I think it is important that readers understand this behavior.

Applies to both instances of this in the PR.